docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -38,18 +44,24 @@ The screen reader will announce the value title (`Number of passengers`) and but
 ```jsx
 <Stack>
   <Stack>
-    <Text id="passengers">Passengers</Text>
+    <Text id="adults-title">Adults</Text>
+    <Text id="adults-description">Number of adult passengers</Text>
   </Stack>
   <Stepper
     step={1}
     minValue={0}
     minValue={10}
     ariaLabelValue="Number of passengers"
-    ariaLabelledby="passengers"
+    ariaLabelledBy="adults-title"
+    ariaDescribedby="adults-description"
     titleDecrement="Remove a passenger"
-    titleIncrement="Add a passenger"
+    titleDecrement="Add a passenger"
+    descriptionIncrement="adults-description"
+    descriptionDecrement="adults-description"
   />
 </Stack>
 ```
 
-This example includes `ariaLabelledby` prop. In this case, `ariaLabelledBy` prop is prioritized over `ariaLabelValue`, so the screen reader will announce the value title (`Passengers`) and buttons title (`Add a passenger`, `Remove a passenger`) once they are focused by the screen reader.
+This example includes `ariaLabelledby` prop. In this case, `ariaLabelledBy` prop is prioritized over `ariaLabelValue`, so the screen reader will announce the value title (`Adults`) and buttons title (`Add a passenger`, `Remove a passenger`) once they are focused by the screen reader.
+
+Value of `ariaDescribedby`, `descriptionIncrement` and `descriptionDecrement` prop is used to announce the complementary information about the Stepper component description (`Number of adults in your group`) once the Stepper input or buttons are focused by the screen reader.
```

Comment:
I think this example is a bit of a mess.

E.g. in the last sentence you mention "Number of adults in your group" but that's not in the example.
Maybe for `descriptionIncrement` and `descriptionDecrement` you wanted to add "adults-title"? Because I don't think that "Add a passenger. Number of adult passengers." is correct 😅 
You also sometimes have `ariaLabelledby` and sometimes `ariaLabelledBy`.
Also I would either select different text because I don't think it makes sense to say for the value "Aduls. Number of adult passengers."

And overall I think it would make sense to write it in order how it's read on focus. So first mention everything that is read for decrement button, then value, then increment button.